### DISCUSSION TITLE
Getting Insufficient data for an Image error after cgm to jpeg conversion in Redhat 7

### DISCUSSION THREAD

**Discussion by bajishaik008** [ImageMagick/ImageMagick#4192] [Help]
We are getting Insufficient data for an Image error after cgm to jpeg conversion in Redhat 7.7.

It is working in Redhat6.

We are converting cgm to jpeg using below command: 
**convert -contrast-stretch 1%x99% -density 300 source.cgm target.jpeg**

After jpeg conversion, we are generating a PDF report(using jrxml) by embedding the  converted jpeg.

Generated PDF is throwing error **"Insufficient data for an image"**

We are able to find the root cause of this issue using -debug all command in convert command. Please check below.
![DebugInfo](https://user-images.githubusercontent.com/4557189/95524811-43f07880-0987-11eb-9adf-71e0ccdce536.png)

Looks like In Redhat 7, Imagemagick is looking for ICC profiles to convert files to jpeg.

**How to fix this issue? Do we need to install any other software for ICC profiles or do we need to change the above command.**

**Environment Details:**
OS - Redhat 7.7
ImageMagick Version - 6.7.8-9
Ralcgm version - 3.50
Ghostscript version - 9.25

**Comment by snibgo** [comment]
Your version of IM is very very old. Does the problem occur in more recent versions?

According to https://en.wikipedia.org/wiki/Computer_Graphics_Metafile , CGM is a format that supports vactor and raster. From your messages I suppose your file contains an embedded PNG raster image, and that contains an embedded ICC profile. The "-debug all" messages do not suggest a problem with this.

I suggest you zip your source.cgm, target.jpeg, and final PDF and upload to somewhere on the internet and paste the URL(s) here.
